workflow/cases/coupled_free_forecast_wave.yaml

```diff
@@ -16,7 +16,7 @@ case:
     print_esmf: .true.
     nems_temp: 'cpld_wave'
     mom6ic_prepared: .false.
-    inline_post: .false. 
+    inline_post: .true. 
```

Comment:
inline post should be false for all cases right now unless inline post updates have been made?

Reply:
This PR also includes updates to inline post functionalities. Let's also test that part.